depenguinme.sh

```diff
@@ -289,8 +291,7 @@ add_drive_to_disks() {
 if [ -n "$USENVME" ]; then
 	echo "Adding NVMe drives..."
 	for i in $(seq 1 "$USENVME"); do
-		# shellcheck disable=SC2086
-		add_drive_to_disks "$(echo $mydrives | sed -n "${i}"p)"
+		add_drive_to_disks "$(echo "$mydrives" | sed -n "${i}"p)"
```

Comment:
This missing quote is important - without it

```
nvme0
nvme1
```

turns into 

```
nvme0 nvme1
```

which means that the sed loop (output line 1...) calls

```
   add_drive_to_disks nvme0 nvme1
```

instead of

```
   add_drive_to_disks nvme0
   add_drive_to_disks nvme1
```

(which means, it only ever adds the first nvme device to disks)

Reply:
That said, I think drives should be handled in a more uniform way (just iterating over the drives array instead of individual lines of output). Would also allow to get rid of the helper function. Should I make that change in here as well?

Reply:
Thanks, will test! I had an issue with the quotes here, that's why it was unquoted and the shellcheck exclusion added, it didn't work as expected when "$mydrives" was quoted.